ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/DeleteItemAction.java

```diff
@@ -173,7 +173,7 @@ protected void onSuccess(final Status result) {
 
                     @Override
                     protected void onFailure(Throwable exception) {
-                        notificationManager.showError(exception.getMessage());
+                        notificationManager.notify("Git", exception.getMessage());
```

Comment:
I don't want to add git dependency from che-plugins only for the single locale constant. We should avoid git operation calls in che-core otherwise.